install.sh

```diff
@@ -13,6 +13,7 @@ SCRIPTS="$MHN_HOME/scripts/"
 cd $SCRIPTS
 
 if [ -f /etc/redhat-release ]; then
+    export PATH=/usr/local/sbin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin
```

Comment:
IMO, you should not do this.  It should be something like:

```
export PATH=/usr/local/sbin:/usr/local/bin:$PATH
```

Resetting the whole PATH may have unintended consequences.